ffs/coreipfs/coreipfs.go

```diff
@@ -5,6 +5,7 @@ import (
 	"context"
```

Comment:
Make a pinset cache. Basically is loaded on first use instead of in `New`. From that point forward, since PG is the only one touching the pinset of the IPFS node it's safe to assume that cache will be always valid and the API call to the IPFS node to get pinsets can be avoided.

Reply:
Sweet.

Reply:
It's an in-memory only thing?

Reply:
Yep, just a guarded `map` of `cid.Cid`.